src/plugins/UserDataPlugin.js

```diff
@@ -54,7 +77,96 @@ export class UserDataPlugin extends BasePlugin {
    * @method onUserDataWrite
    * @private
    */
-  onUserDataWrite({ type, data: { name, value } }) {
-    SavedDataHandler.write(name, value, () => this.client.send(type));
+  onUserDataWrite({type, data: { name, value } }) {
+
+    SavedDataHandler.write(name, value,  () => this.client.send(type));
+  }
+
+  // ----------------------------------------------------------------
+  //                      IndexedDB Manipulation        
+  // ----------------------------------------------------------------
+
+  /**
+   * Open a connection with the IDB Database and optionally add or delete
```

Comment:
Object derestructuring parameters can be a little weird in JSDOC comments. 
See:
https://stackoverflow.com/questions/36916790/document-destructured-function-parameter-in-jsdoc
For some examples, but in general you want to have it like:
`@param {Object} params`
`@param {string} params.type whatever`
Then you'd also want 
`@param {Object} params.data`
and so on